src/formio/components/licenseplate.ts

```diff
@@ -0,0 +1,35 @@
+import {InputComponentSchema, MultipleCapable} from '..';
+
+type Validator = 'required';
```

Comment:
I think we need to add `pattern` so that we can allow translations for the validation error message:

```suggestion
type Validator = 'required' | 'pattern';
```

but in the form builder, we will only set the 'default value' for `validate.pattern` and not expose a form control.